stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Message.kt

```diff
@@ -123,3 +125,25 @@ private fun getAttachmentPrefix(attachment: Attachment): String? =
         AttachmentType.GIPHY -> "/giphy"
         else -> null
     }
+
+internal fun Message.getTranslatedText(currentUser: User?): String {
+    return when (ChatUI.autoTranslationEnabled) {
+        true -> currentUser?.language?.let { userLanguage ->
+            getTranslation(userLanguage).ifEmpty { text }
+        } ?: text
+        else -> text
+    }
```

Comment:
```suggestion
    return getTranslatedText { currentUser }
```

Reply:
Nice catch @JcMinarro 👍 
Thank you ❤️

Reply:
I've also made the the following function as `inline` one to reduce the overhead 👍 
```kotlin
internal inline fun Message.getTranslatedText(getCurrentUser: () -> User?): String
```